packages/eas-cli/src/build/metadata.ts

```diff
@@ -51,7 +52,7 @@ export async function collectMetadataAsync<T extends Platform>(
     workflow: ctx.workflow,
     credentialsSource: ctx.buildProfile.credentialsSource,
     sdkVersion: ctx.exp.sdkVersion,
-    runtimeVersion: ctx.exp.runtimeVersion,
+    runtimeVersion: Updates.getRuntimeVersion(ctx.exp, ctx.platform),
```

Comment:
https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/build/ios/UpdatesModule.ts#L39

For generic projects, we are syncing runtimeVersion from app.json with Expo.plist file. Are there any changes required there?

Reply:
We should be fine.

Both `isPlistConfigurationSynced` and `setVersionsConfig` call into `getRuntimeVersionNullable`: 
https://github.com/expo/expo-cli/blob/91549debf9ec555562a2aee8ada76304cc0733d2/packages/config-plugins/src/ios/Updates.ts#L200
https://github.com/expo/expo-cli/blob/91549debf9ec555562a2aee8ada76304cc0733d2/packages/config-plugins/src/ios/Updates.ts#L92

Which is a wrapper around this same function `getRuntimeVersion`